confgenerator/filter/internal/ast/ast.go

```diff
@@ -0,0 +1,435 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ast
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/filter/internal/generated/token"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
+)
+
+type Attrib interface{}
+
+type Target []string
```

Comment:
Please duplicate the comment about this being `member` here. (Or keep the BNF as `member` for consistency with upstream and put the comment only here.)

Reply:
The latter is more sensible to me so I'll do that.